src/main/java/com/baran/charactersheetcreator/configuration/UserDetails.java

```diff
@@ -0,0 +1,29 @@
+package com.baran.charactersheetcreator.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.crypto.factory.PasswordEncoderFactories;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.provisioning.InMemoryUserDetailsManager;
+
+@Configuration
+public class UserDetails {
```

Comment:
Just as a note:
In a productive context these users mustn't be created, while in a dev setup they are very useful.  
How could this be accomplished?

Reply:
You could accomplish this in productive context with an Identity and Access Management Software like keycloak.
But even though i have ideas why you shouldnt create themeselves in productive context, im not shure why you mustnt'. Please explain that to me next time we see us :D